doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -162,7 +162,9 @@ nRF9160: Serial LTE modem
 nRF5340 Audio
 -------------
 
-|no_changes_yet_note|
+* Added:
+
+  * Can now set periodic and extended advertising interval through Kconfig.
```

Comment:
```suggestion
* Added  Kconfig options for setting periodic and extended advertising intervals.
  Search :ref:`Kconfig Reference <kconfig-search>` for ``BLE_ACL_PER_ADV_INT_`` to list all of them.
```

Reply:
There is also BLE_ACL_EXT_ADV_INT_
Remove double space after "Added"

Reply:
Yes, "Added:" is needed for more than 1 item, to avoid repeating "Added" in front of each line and signal clearly with indents what entries belong where.